contracts/GuestList.vy

```diff
@@ -43,6 +51,25 @@ def set_bouncer(new_bouncer: address):
     self.bouncer = new_bouncer
 
 
+@view
+@internal
+def _total_yfi(user: address) -> uint256:
+    return (
+        self.yfi.balanceOf(user)
+        + self.ygov.balanceOf(user)
+        + self.yyfi.balanceOf(user) * self.yyfi.getPricePerFullShare() / 10 ** 18
+    )
+
+
+@view
+@external
+def total_yfi(user: address) -> uint256:
+    """
+    Total YFI in wallet + yGov + YFI vault.
+    """
+    return self._total_yfi(user)
+
+
 @view
 @external
 def authorized(guest: address, amount: uint256) -> bool:
```

Comment:
no longer needs ` amount` arg or we want to keep signature as is?

Reply:
This is the interface in the current Vault version. I'll add the default amount so Vyper generates both `authorized(address,uint256)` and `authorized(address)` in case we change that.